apps.yaml

```diff
@@ -220,3 +220,15 @@ HierCVAE:
         - inverse-design
         - ml-metadata
         - wp11
+tomato:
+    metadata:
+        title: tomato
+        description: |
+            tomato: au-tomation without the pain. Instrument automation library. Part of the dgbowl suite of tools for digital (electro-)catalysis and battery materials research.
+        authors: Peter Kraus
+        documentation_url: https://dgbowl.github.com/tomato
+        logo: https://dgbowl.github.io/images/tomato.png
+        state: development
+    git_url: https://github.com/dgbowl/tomato
+    categories:
+        - technology-lab
```

Comment:
Please add the categories proposed in the PR comments and then add them here. Thank you!